src/main/java/frc/team166/robot/subsystems/Cart.java

```diff
@@ -80,4 +80,28 @@ protected void end() {
             }
         };
     }
+
+    public Command DriveStraight() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+
+            }
+
+            @Override
+            protected void execute() {
+
+            }
+
+            @Override
+            protected boolean isFinished() {
+                return false;
+            }
+
+            @Override
+            protected void end() {
+                drivePidController.disable();
+            }
+        };
+    }
 }
```

Comment:
Why are we still including the cart in the robot?

Reply:
This has finally been removed, they just need to rebase/merge master in.